### PR TITLE
Do not run tests on java 9

### PR DESCRIPTION
**Pull request by GaneshSPatil**
* Remove java 9 from supported JVM

### CODE REVIEW

File: .ci/with-java [hunk 1]
```diff
@@ -13,10 +13,8 @@ if [ "$mod" = "0" ]; then
   # use the system jvm
   true
 elif [ "$mod" = "1" ]; then
-  use_jdk "openjdk@1.9.0-4" "tgz+https://nexus.gocd.io/repository/s3-mirrors/local/jdk/openjdk-9.0.4_linux-x64_bin.tar.gz"
-elif [ "$mod" = "2" ]; then
   use_jdk "openjdk@1.10.0-2" "tgz+https://nexus.gocd.io/repository/s3-mirrors/local/jdk/openjdk-10.0.2_linux-x64_bin.tar.gz"
-elif [ "$mod" = "3" ]; then
+elif [ "$mod" = "2" ]; then
   use_jdk "openjdk@1.11.0-28" "tgz+https://nexus.gocd.io/repository/s3-mirrors/local/jdk/openjdk-11-28_linux-x64_bin.tar.gz"
```
Comment (bdpiparva) [inline .ci/with-java]: Can we use `jabba use openjdk@1.11.0-28` instead? That will fail the build if java is not installed on container.

Comment (GaneshSPatil) [reply .ci/with-java]: See method `use_jdk`.

**Comment by ketan** [comment]
There's also a powershell script that you've missed out on?

**Comment by GaneshSPatil** [comment]
> There's also a powershell script that you've missed out on?

FIxed!

**Comment by GaneshSPatil** [comment]
Do not merge! We should remove jdk9 and add jdk12 to the support metrix

**Comment by bdpiparva** [comment]
@GaneshSPatil - Sure, we can target support for java11 in next release and also give it a try to java12.

**Pull request merged**